terms.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Terms of Use"
+---
+
+## 1. Introduction
+
+These Terms of Use (the "Terms") govern your use of R-multiverse (the "Repository"). By using the Repository, you agree to comply with these Terms. If you do not agree with these Terms, you may not use the Repository. 
+
+## 2. No Warranty
+
+**AS-IS BASIS**: All packages and services provided through the Repository are offered "as-is," without any warranty, either express or implied. This includes, but is not limited to, implied warranties of merchantability, fitness for a particular purpose, non-infringement, and accuracy of data.
+
+Neither the R-Multiverse Management Committee (as defined in the [R-multiverse Governance Document](governance.md)), nor any Administrator or Moderator of the Repository, nor any Contributor to the Repository (collectively, "the Contributors") shall be held responsible for any defects in the packages or any damages that result from their use.
```

Comment:
should the link be to a rendered version not the md?

Reply:
Quarto allows this, so the rendered version will link to the html. I don't know what is best practice here.

Reply:
oooh that's awesome